librz/asm/arch/include/mybfd.h

```diff
@@ -1506,7 +1506,9 @@ extern const struct bfd_symbol *const bfd_ind_symbol;
 			(struct bfd_symbol **)SYM_PTR, \
 \
 			/* map_head, map_tail                                            */ \
-			{ NULL }, { NULL } \
+			{ NULL }, { \
```

Comment:
This one looks weird.

Reply:
I agree. Should I change the `.clang-format`?

Reply:
Do you know what would change this behavior? It might be just a one-off case.

Reply:
No, I do not know. I'll have a look at the documentation and the changelog.

Reply:
I am unable to find the option which can make these blocks look as they did in clang-format 13.

Changelog: https://releases.llvm.org/14.0.0/tools/clang/docs/ReleaseNotes.html#clang-format
Documentation: https://clang.llvm.org/docs/ClangFormatStyleOptions.html

Reply:
You can just add off/on comments

Reply:
Maybe it also makes sense to open a bug in clang-format itself, it does look like it

Reply:
I'll probably ask first to the folks at clang-format, and see if it is really not possible to do this. If it indeed is not possible, we can add off/on directives till clang-format fixes it.